docs/using.rst

```diff
@@ -188,6 +190,24 @@ expected output is::
 .. _README: ../components/centos/centos-k8s-singlenode-setup/README.rst
 .. _Vagrantfile: ../components/centos/centos-k8s-singlenode-setup/Vagrantfile
 
+Vagrant 2 way folder sync
+=========================
+For basic usage please refer `Vagrant documentation. <https://www.virtualbox.org/manual/ch04.html#sharedfolders>`_
+
+In some scenarios 2 way bi-directional sync makes more sense as it is the simplest way to move files (e.g. code) between 
```

Comment:
@hferentschik I do not agree. I think Vagrant documentation talks about one way copy of a directory to the guest as the standard behavior e.g. https://www.vagrantup.com/docs/synced-folders/ . Also in Vagrant ecosystem it is understood that guest is ephemeral so , it is not recommenced to keep things inside the guest.